.github/workflows/legal-api-ci.yml

```diff
@@ -83,6 +83,10 @@ jobs:
         # needed because the postgres container does not provide a healthcheck
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
This may be required for the step on line 95 to work correctly.

This same step works fine in the linting job so trying it here in the testing job.